crates/agglayer-tries/src/utils.rs

```diff
@@ -1,15 +1,1000 @@
-use agglayer_primitives::{keccak::keccak256_combine, Digest};
+use agglayer_primitives::Digest;
 
 /// Returns an array whose `i`th element is the root of an empty Merkle tree of
 /// depth `i`.
 #[inline]
-pub fn empty_hash_at_height<const DEPTH: usize>() -> [Digest; DEPTH] {
+pub const fn empty_hash_at_height<const DEPTH: usize>() -> [Digest; DEPTH] {
```

Comment:
I vote to change the name of the function to `empty_hash_array_with_height`. This way is a bit misleading.